optics-core/src/Optics/Internal/Indexed.hs

```diff
@@ -61,7 +60,7 @@ instance
   ) => CheckIndices i '[i1, i2, i3, i4, i5]
 
 instance
-  ( TypeError ('Text "Use icompose* variants to flatten indices")
+  ( TypeError ('Text "Use icomposeN to flatten indices")
```

Comment:
we don't export `icomposeN`, do we.

I think we could, but then we have to export the `CurryCompose` family (or at least mention that it has instances for all type level lists).

Reply:
Alright, I reverted this change to error message for now, we can do that later.